settings.gradle

```diff
@@ -147,18 +144,27 @@ project(':client-session').projectDir = file('java-client/session')
 include(':client-flight')
 project(':client-flight').projectDir = file('java-client/flight')
 
+include(':client-deephaven')
+project(':client-deephaven').projectDir = file('java-client/deephaven')
+
 include(':client-session-dagger')
 project(':client-session-dagger').projectDir = file('java-client/session-dagger')
 
 include(':client-flight-dagger')
 project(':client-flight-dagger').projectDir = file('java-client/flight-dagger')
 
+include(':client-deephaven-dagger')
+project(':client-deephaven-dagger').projectDir = file('java-client/deephaven-dagger')
+
 include(':client-session-examples')
 project(':client-session-examples').projectDir = file('java-client/session-examples')
 
 include(':client-flight-examples')
 project(':client-flight-examples').projectDir = file('java-client/flight-examples')
 
+include(':client-deephaven-examples')
+project(':client-deephaven-examples').projectDir = file('java-client/deephaven-examples')
```

Comment:
I think we need to figure out a better name. `client-barrage`? `client-worker`? `client-full`?